REST_API_DOCS.md

```
@@ -0,0 +1,1289 @@
+
+| 헤더            | 타입     | 필수 | 설명        |
+|---------------|--------|----|-----------|
+| Authorization | string | O  | Bearer 토큰 |
+
+**Path Parameters**
+
+| 파라미터    | 타입     | 필수 | 설명              |
+|---------|--------|----|-----------------|
+| post_id | string | O  | 좋아요를 등록할 게시글 ID |
+
+**Response (201 Created)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)